src/main/java/janet/Ui.java

```diff
@@ -0,0 +1,167 @@
+package janet;
+
+/**
+ * Represents the UI of Janet.
+ */
+public class Ui {
+    // handles the interaction with the user
+    // receives inputs (readline) and printing outputs to the command line
+
+    private static final String HORIZONTAL_LINE = "____________________________________________________________";
+
+    public void showLine() {
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints the welcome message.
+     */
+    public void showWelcome() {
+        System.out.println(HORIZONTAL_LINE + "\nHello! I'm Janet\n" + "What can I do for you?\n" + HORIZONTAL_LINE);
+    }
+
+
+    /**
+     * Prints the exit message.
+     */
+    public void exitMessage() {
```

Comment:
I think you can consider naming this method with an action verbs instead, like `showExitMessage` or `displayExitMessage`.

Reply:
Good point! I must have missed that out